src/js/modules/SelectRange/SelectRange.js

```diff
@@ -873,6 +874,7 @@ export default class SelectRange extends Module {
 	
 	
 	getActiveCell() {
+		if(!this.activeRange) return;
```

Comment:
Since this could be `undefined`, could you make sure that other places that use `getActiveCell()` handles `undefined` properly?

Reply:
It is used in only two places. One already checked the return value and to the other one I  added an check.